### PR TITLE
add money to practical supports

### LINKED ISSUE #2415: Add concept of money to practical support object
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**

add an integer column for money on the practical support object

* **What feature or behavior is this required for?**

* **How could we solve this issue? (Not knowing is okay!)**

* **Anything else?**

design https://www.figma.com/proto/DjNqW3YKhAfXIkd7SCLmzL/Practical-Support?node-id=138%3A[…]ng=min-zoom&page-id=79%3A305&starting-point-node-id=138%3A3878

### PR DESCRIPTION
**Pull request by elimbaum**
I rule and have completed some work on Case Manager that's ready for review!

(brief, plain english overview of your changes here)

This pull request makes the following changes:
* (your change here)
* (another change here)
* (etc)

(If there are changes to the views, please include a screenshot so we know what to look for!)

It relates to the following issue #s: 
* Fixes #2415 
* Bumps #Y

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

**Comment by elimbaum** [comment]
Here's what the draft UI looks like:

Screen Shot 2021-11-17 at 21 52 23

Support type is no longer a dropdown, and we add Amount number-fields.

**Comment by elimbaum** [comment]
Inconsistent test results here - system tests seemingly unrelated to what I've done are occasionally failing. May be something to do with the system-test parallelization?

File: app/models/practical_support.rb [hunk 1]
```diff
@@ -11,4 +11,7 @@ class PracticalSupport < ApplicationRecord
   # Validations
   validates :source, :support_type, presence: true
   validates :support_type, uniqueness: { scope: :can_support }
+  validates :amount, 
+              allow_nil: true,
+              numericality: { only_integer: true, greater_than: 0 }
```
Comment (colinxfleming) [inline app/models/practical_support.rb:16]: 🙏  great call here

**Review by colinxfleming — APPROVED** [review]
will fire shortly. thanks much for knocking this out so quick and staying on it!

**Comment by colinxfleming** [comment]
fyi to @sarahcoloma  - this is going in now and we'll deploy it around the weekend most likely!

**Pull request merged**